app/javascript/dashboard/helper/specs/downloadHelper.spec.js

```diff
@@ -1,24 +1,4 @@
-import { downloadCsvFile, generateFileName } from '../downloadHelper';
-
-const fileName = 'test.csv';
-const fileData = `Agent name,Conversations count,Avg first response time (Minutes),Avg resolution time (Minutes)
-Pranav,36,114,28411`;
-
-describe('#downloadCsvFile', () => {
-  it('should download the csv file', () => {
-    const link = {
-      click: jest.fn(),
-    };
-    jest.spyOn(document, 'createElement').mockImplementation(() => link);
```

Comment:
Removed this test as `createObjectURL` is not testable with the version of jest we are using. Stubbing createObjectURL doesn't make sense as it is core of the method.

Reply:
https://stackoverflow.com/a/52969731